README.md

```diff
@@ -19,55 +19,55 @@
 - <a href="#about-spin">About Spin</a>
 - <a href="#the-multiple-ways-to-build-a-spin">The multiple ways to build a Spin</a>
 - <a href="#the-multiple-ways-to-create-a-feedback">The multiple ways to create a Feedback</a>
-- <a href="#feedbacks-lifecycle">Feedbacks lifecycle</a>
+- <a href="#feedback-lifecycle">Feedback lifecycle</a>
 - <a href="#feedbacks-and-scheduling">Feedbacks and scheduling</a>
 - <a href="#using-spin-in-a-uikit-or-appkit-based-app">Using Spin in a UIKit or AppKit based app</a>
 - <a href="#using-spin-in-a-swiftUI-based-app">Using Spin in a SwiftUI based app</a>
 - <a href="#demo-applications">Demo applications</a>
 - <a href="#acknowledgements">Acknowledgements</a>
 
 
-# About State machines
+# About State Machines
 
-**What is a State machine ?**
+**What is a State Machine?**
 
-> It is an abstract machine that can be in exactly one of a finite number of states at any given time. The state machine can change from one state to another in response to some external inputs. The change from one state to another is called a transition. A state machine is defined by a list of its states, its initial state, and the conditions for each transition
+> It's an abstract machine that can be in exactly one of a finite number of states at any given time. The state machine can change from one state to another in response to some external inputs. The change from one state to another is called a transition. A state machine is defined by a list of its states, its initial state, and the conditions for each transition
 
-Guess what ? An application IS a state machine.
+Guess what! An application IS a state machine.
 
 We just have to find the right tool to implement it. This is where feedback loops come into play 👍.
 
-A Feedback loop is a system that is able to self regulate by giving the resulting value from its computations as the next input to itself, constantly adjusting this value according to given rules (Feedback loops are used in domains like electronics to automatically adjust the level of a signal for instance).
+A Feedback Loop is a system that is able to self-regulate by using the resulting value from its computations as the next input to itself, constantly adjusting this value according to given rules (Feedback Loops are used in domains like electronics to automatically adjust the level of a signal for instance).
```

Comment:
If Feedback is capitalized, I was wondering if "Loop" should also be since they go together.

Reply:
Look good to me with a L